README.md

```diff
@@ -1,16 +1,189 @@
+
+Methods that behave in this way we'll say have the "Character of Collection
+Processing."
+
+In this module, we'll learn to use JavaScript's "Collection-Processing Methods"
+to answer the type of question which involves "polling each member."
+
+> **USAGE NOTE** In Ruby, all the methods that behave like this are called
+> "Enumerable." There's no official collective term for methods that do this
+> "polling each member" operation. In the rest of this module, we'll call them
+> "Collection-Processing Methods." This isn't a hard-and-fast term that
+> programmers use in daily practice, but all JavaScript programmers know that
+> there's something similar to the methods we're about to introduce, so nobody
+> would look at you too strangely for speaking of "JavaScript's Collection
+> Methods."
+
+## Pseudocode a Real-World Use of "Collection-Processing Methods"
+
+Let's consider how a real-world need might drive us to think and behave in
+accordance with the "Character of Collection Processing."
+
+True story: someone close to us got married last year. In anticipation of his
+destination wedding and fabulous honeymoon in Italy, he was determined to
+reduce his chances of getting sick. The only problem, he takes public transit
+every day to work.
+
+His **Question** was: "How can I avoid being sick for these special occasions?"
+
+Doing some research, he realized he could reduce airborne infection risk by
+_wearing a surgical mask_. But he didn't want to wear one all the time so he
+decided to implement the following modification to his behavior:
+
+> **PSEUDOCODE**: Below we're going to use "pseudocode." It's something that
+> looks a bit like code, but we're not expecting that it would run, it's just a
+> convenient way to express a problem's solution in a way like code, but far
+> less demanding. It's common to find programmers "sketching" a problem or a
+> phenomenon in pseudocode because it's usually shorter to write than English
+> and, once you learn programming, it's a handy way to communicate with other
+> programmers.
+
+```ruby
+while on_the_train
+  if i_hear_a_sick_sound
+    my_mask_status = true
+  end
+end
+```
+
+The method name `i_hear_a_sick_sound`'s pseudocode looks like
+this:
+
+```ruby
+def i_hear_a_sick_sound(passengers)
+  # Given a collection of passengers' sounds ["coughing", "yawning", "sneezing", "singing Jamaican traditional folksong"]
+  # If any of them are sick sounds: cough, yawn
+  # return `true`; else, return `false`
+end
+```
+
+If we were to encode this, we might write the (real code, not pseudocode) the
+method as:
+
+```js
+function iHearASickSound(passengersSounds) {
+  let heardSickSound = false
+  for (let i = 0; i < passengersSounds.length; i++) {
+    if (passengersSounds[i] === "coughing" ||
+        passengersSounds[i] === "sneezing" ) {
+      return true
+    }
+  }
+  return false;
+}
+
+console.log(iHearASickSound(["coughing", "foo", "bar", "bin", "bat"])) //=> true
```

Comment:
logs may not be necessary here. in node and browser, these function calls will still just return true